netty/src/main/java/io/grpc/netty/WriteQueue.java

```diff
@@ -101,6 +101,19 @@ void enqueue(Runnable runnable, boolean flush) {
     }
   }
 
+  /**
+   * Execute enqueued work directly on the current thread. This can be used to trigger writes before
```

Comment:
s/Execute/Executes/

Reply:
Done. Hmm... the other methods are wrong as well. I'll leave them as-is.